### DISCUSSION TITLE
point_0719

### DISCUSSION THREAD

**Discussion by andrea29-star** [cloudsen12/dataset#279] [Roy]
"20191213T075229_20191213T075529_T37QED"
![image](https://user-images.githubusercontent.com/54947195/147612208-6c8ed421-e7d7-419a-b60e-cca32899e407.png)

"20190909T074611_20190909T075348_T37QED"
![image](https://user-images.githubusercontent.com/54947195/147612334-8bf92f2c-c9b3-445f-8366-1466a6b27b7d.png)
![image](https://user-images.githubusercontent.com/54947195/147612382-27c239a0-c20b-4a43-8c25-8d668902894f.png)

"20190810T074621_20190810T075719_T37QED"
![image](https://user-images.githubusercontent.com/54947195/147612439-a0eba184-9793-42f1-95ce-d0763e930ee1.png)

"20181228T075329_20181228T080048_T37QED"
![image](https://user-images.githubusercontent.com/54947195/147612473-91add074-8288-4088-b7ff-20a59d04bb72.png)

@ryali93

**Comment by AntoniCesar** [comment]
"20191213T075229_20191213T075529_T37QED"
![Captura](https://user-images.githubusercontent.com/86502862/153534981-1a201faf-bae5-4cef-95c5-44e7fe6c35b4.JPG)

**Comment by AntoniCesar** [comment]
"20190909T074611_20190909T075348_T37QED"
![Captura](https://user-images.githubusercontent.com/86502862/153539920-99318fd8-af86-429b-8cd6-1445cc922d96.JPG)
